### DISCUSSION TITLE
Which compiler is implied for "Compatibility: ✓ x64-windows"?

### DISCUSSION THREAD

**Discussion by felixf4xu** [microsoft/vcpkg#38438] [Q&A]
Hi,
![image](https://github.com/microsoft/vcpkg/assets/84662027/3b19a671-91f5-40ab-bfce-2f09b1abd39f)

When it says "Compatibility: ✓ x64-windows", which compiler is implied? MSVC? or all compilers? or at least one of MSVC and GCC/Clang?

**Comment by dg0yt** [comment]
`x64-windows` is a vcpkg triplet which unambiguously means MSVC.

YMMV: These compatiblity hints on derived websites have a very limited perspective: the `supports` field in the manifest. Actual compatibility may be limited by dependencies (vcpkg and external). Support for unofficial triplets is more or less unknown at this point.
